src/styles/index.scss

```diff
@@ -1,3 +1,146 @@
 body {
-  margin: 0;
+  $color-black: #060b35;
+  $color-white: #fff;
+  $color-gray: #616070;
+  $color-blue: #00acdc;
+  $gap: 4px;
+
+  body,
+  p,
+  a,
+  img,
+  div {
+    font-family: Roboto, Helvetica, sans-serif;
+    margin: 0;
+    box-sizing: border-box;
+  }
+
+  .card {
+    width: 200px;
+    height: 406px;
+    padding: 32px 16px 16px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+
+    &__image {
+      display: block;
+      width: 160px;
+      height: 134px;
+      margin: 0 auto;
+    }
+
+    &__title {
+      font-style: normal;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 500;
+      margin-top: 40px;
+      margin-bottom: 0;
+      color: $color-black;
+      overflow: hidden;
+    }
+
+    &__code {
+      font-style: normal;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      margin-top: 4px;
+      color: $color-gray;
+    }
+
+    &__rating {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-top: 16px;
+    }
+
+    &__reviews {
+      font-style: normal;
+      font-size: 10px;
+      font-weight: 400;
+    }
+
+    &__cost {
+      display: flex;
+      justify-content: space-between;
+      height: 18px;
+      margin-top: 24px;
+
+    }
+
+    &__price {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $color-gray;
+    }
+
+    &__amount {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $color-black;
+    }
+
+    &__link {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      font-size: 14px;
+      font-weight: 700;
+      text-transform: uppercase;
+      text-decoration: none;
+      width: 100%;
+      height: 40px;
+      margin-top: 16px;
+      color: $color-white;
+      border-radius: 5px;
+      background-color: $color-blue;
+      border: 1px solid transparent;
```

Comment:
Please, check and follow this approach [css-rule-order](https://9elements.com/css-rule-order/)

Reply:
done :)